apps/remix-ide/src/app/components/plugin-manager-component.js

```diff
@@ -84,112 +22,89 @@ const profile = {
 class PluginManagerComponent extends ViewPlugin {
   constructor (appManager, engine) {
     super(profile)
-    this.event = new EventEmitter()
     this.appManager = appManager
+    this.engine = engine
+    this.pluginManagerSettings = new PluginManagerSettings()
+    this.htmlElement = document.createElement('div')
+    this.htmlElement.setAttribute('id', 'pluginManager')
     this.views = {
```

Comment:
Is this line still relevant? If not, remove it.

Reply:
Removed.